codegen.yml

```diff
@@ -8,9 +8,13 @@ generates:
       - 'typescript-operations'
```

Comment:
changes in this file allow automatically typing BigNumber values as BigNumber (by default these are typed `any`)